### PR TITLE
adds content protection information fixes #48

### LINKED ISSUE #48: Adds content protection information

### PR DESCRIPTION
**Pull request by leandromoreira**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -736,11 +737,60 @@ HEVC has bigger and more **partitions** (and **sub-partitions**) options than AV
...
+#### Main systems
+
+* FPS - [**FairPlay Streaming**](https://developer.apple.com/streaming/fps/)
+* PR - [**PlayReady**](https://www.microsoft.com/playready/)
+* WV - [**Widevine**](http://www.widevine.com/)
+
+
+#### What?
+
+DRM means Digital rights management, it's a way **to provide copyright protection for digital media**, for instance, digital video and audio. Although it's used in many places [it's not universally accepted](https://en.wikipedia.org/wiki/Digital_rights_management#DRM-free_works).
```
Comment (danielfm) [inline README.md]: Capitalization fix:

> DRM means Digital Rights Management, ...

File: README.md [hunk 2]
```diff
@@ -736,11 +737,60 @@ HEVC has bigger and more **partitions** (and **sub-partitions**) options than AV
...
+
+> **<sup>1</sup>protected**: this protection can be **via hardware**, for instance, this key can be stored inside a special (read-only) chip that works like [a black-box](https://en.wikipedia.org/wiki/Black_box) to provide decryption, or **by software** (less safe), the DRM system provides means to know which type of protection a given device has.
+
+
+When the **player P1 wants to play** the **content C'1**, it needs to deal with the **DRM system DRM1**, giving its public key **PUK1**. The DRM system DRM1 returns the **key K1 encrypted** with the client''s public key **PUK1**. In theory, this response is something that **only D1 is capable of decrypting**.
+
+`K1P1D1 = enc(K1, PUK1)`
+
+**P1** uses its DRM local system (it could be a [SOC](https://en.wikipedia.org/wiki/System_on_a_chip), a specialized hardware or software), this system is **able to decrypt** the content using its private key PRK1, it can decrypt **the symmetric-key K1 from the K1P1D1** and **play C'1**. At best case, the keys are not exposed through memory RAM.
```
Comment (danielfm) [inline README.md]: > At best case, the keys are not exposed through ~memory~ RAM.

File: README.md (hunk 1 above)
Comment (danielfm) [inline README.md]: Disregard this comment, it's actually correct according to Google 😅

**Comment by leandromoreira** [comment]
Thank you very much @danielfm

**Pull request merged**